mmv1/third_party/terraform/services/compute/resource_compute_interconnect_macsec_test.go.tmpl

```diff
@@ -0,0 +1,185 @@
+  noc_contact_email    = "user@example.com"
+  requested_features   = ["IF_MACSEC"]
+  labels = {
+    mykey = "myvalue"
+  }
+}
+`, context)
+}
+
+func testAccComputeInterconnect_computeInterconnectEnableMacsec(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "project" {}
+
+resource "google_compute_interconnect" "example-interconnect" {
+  name                 = "tf-test-example-interconnect%{random_suffix}"
+  customer_name        = "internal_customer" # Special customer only available for Google testing.
+  interconnect_type    = "DEDICATED"
+  link_type            = "LINK_TYPE_ETHERNET_100G_LR"
+  location             = "https://www.googleapis.com/compute/v1/projects/${data.google_project.project.name}/global/interconnectLocations/z2z-us-east4-zone1-lciadl-z" # Special location only available for Google testing.
+  requested_link_count = 1
+  admin_enabled        = true
+  description          = "example description"
+  macsec_enabled       = true
+  noc_contact_email    = "user@example.com"
+  requested_features   = ["IF_MACSEC"]
+  macsec {
+    pre_shared_keys {
+      name = "test-key"
+      start_time = "2023-07-01T21:00:01.000Z"
+    }
+    fail_open = true
+  }
+  labels = {
+    mykey = "newvalue"
+  }
+}
+`, context)
+}
+
+{{ if ne $.TargetVersionName `ga` -}}
+func TestAccComputeInterconnect_computeInterconnectCrossSiteNetworkTest(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      "z2z-us-west8-zone2-tzphxd-z",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckComputeInterconnectDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeInterconnect_crossSiteInterconnectBasic(context),
+			},
+			{
+				ResourceName:            "google_compute_interconnect.example-interconnect-cross-site-network",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "location", "terraform_labels"},
+			},
+			{
+				Config: testAccComputeInterconnect_crossSiteInterconnectUpdate(context),
+			},
+			{
+				ResourceName:            "google_compute_interconnect.example-interconnect-cross-site-network",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "location", "terraform_labels"},
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_compute_interconnect.example-interconnect-cross-site-network", "description", "example description updated")),
+			},
+		},
+	})
+}
+
+func testAccComputeInterconnect_crossSiteInterconnectBasic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "project" {
+provider    = google-beta}
```

Comment:
This is causing the test to fail to even run. (Of course once it runs it will have the same problem as the other test).

```suggestion
  provider    = google-beta
}
```